soroban-sdk/Cargo.toml

```diff
@@ -61,8 +62,15 @@ sha2 = "0.10.7"
 [features]
 alloc = []
 testutils = ["soroban-sdk-macros/testutils", "soroban-env-host/testutils", "soroban-ledger-snapshot/testutils", "dep:ed25519-dalek", "dep:arbitrary", "dep:derive_arbitrary", "dep:ctor", "dep:soroban-ledger-snapshot"]
-hazmat = []
 docs = []
 
+# Umbrella feature that enables all hazmat sub-features (backwards compatible)
+hazmat = [
```

Comment:
Are we planning to add hazmat-address here as well? If so, it seems like it would be quite easy to pick up all the new hazmat features as they come - maybe it should be reserved as a legacy alias for hazmat-crypto?

Reply:
Do you mean have `hazmat` as an alias only for crypto? Yeah that's not a bad idea. I'm also thinking we just get rid of it too in the next major release.

Reply:
Right, that would be the plan. Make a non-breaking change for the minor release, but only use it for crypto for compatibility. Then make a breaking change in the major release. This way users of the address functionality won't be affected at all.

Reply:
I'll keep it as an umbrella for now we can still delete it at the next major.